test/features/support/Fixtures.js

```diff
@@ -2,6 +2,10 @@ const Fixtures = function Fixtures() {
   this.sampleJob = {
     title: 'Test job',
     employer: 'Test employer',
+    sourceType: 'online',
+    sourceUrl: 'http://blog.queensland.com/best-of/best-job-in-the-world-best-of/',
+    rating: 4,
```

Comment:
if `Fixtures` is explicitly meant to be canned form data then this should be a string